.github/workflows/ci.yml

```diff
@@ -96,9 +96,28 @@ jobs:
         env:
           TAG: ${{ steps.vars.outputs.tag }}
           CMD: ${{ inputs.test_cmd }}
-          AUTH_SECRET: ${{secrets.gcp_service_account}}
+          AUTH_SECRET: ${{ secrets.gcp_service_account }}
         run: |
-          docker run -v $GOOGLE_APPLICATION_CREDENTIALS:$GOOGLE_APPLICATION_CREDENTIALS -e GOOGLE_APPLICATION_CREDENTIALS -e CI=true -e AUTH_SECRET gcr.io/$GITHUB_REPOSITORY:$TAG $CMD
+          docker run \
+            --name ${{ github.sha }} \
```

Comment:
@arikmaor will adding the `name` option to the `docker run` command change anything?